packages/atomic/src/chip/LocalizeChip.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import classnames from 'classnames';
+import { lighten } from 'polished';
+
+import {
+  LocalizeThemeProps,
+  LocalizeProps,
+  LocalizeSize,
+} from '@seolhun/localize-components-styled-types';
+
+const DEFAULT_CLASSNAME = '__Localize__Button';
+type SpanProps = React.HTMLAttributes<HTMLSpanElement>;
+
+export interface LocalizeChipProps extends LocalizeProps, SpanProps {
+  size?: LocalizeSize;
+
+  borderRadius?: string;
+}
+
+const getSizeStyle = (size?: LocalizeSize) => {
+  switch (size) {
+    case 'xl':
+      return '1.4rem 2rem';
+    case 'lg':
+      return '1.2rem 1.8rem';
+    case 'md':
+      return '1rem 1.4rem';
+    case 'sm':
+      return '0.8rem 1rem';
+    default:
+      return '0.6rem 0.8rem';
+  }
+};
+
+const StyledLocalizeChip = styled.span<LocalizeChipProps, LocalizeThemeProps>(
+  ({ theme, size = 'md', bgColor = 'primary', borderRadius }) => {
+    const fonts = theme.fonts.font1;
+    const backgroundColor = theme.colors[bgColor];
```

Comment:
Codacy found an issue: [Variable Assigned to Object Injection Sink](https://app.codacy.com/manual/shun10114/localize-components/pullRequest?prid=5930444)